src/App.vue

```diff
@@ -307,6 +315,14 @@ export default Vue.extend({
         this.$store.commit("setUnauthorizedError", null);
       }
     },
+    isPasswordResetSuccess: {
+      get() {
+        return this.$store.state.passwordResetSuccess;
+      },
+      set() {
+        this.$store.commit("setPasswordResetSuccess");
```

Comment:
This is too strange. v-snackbar will attempt to set `isPasswordResetSuccess = false`, which will in turn commit passwordResetSuccess=true to store.

Does the notification even ever disappear?

Reply:
Yes, the notification disappears in 9 seconds, but you are right, the logic was strange, it fixed now

Reply:
Just because this was strange to me, I looked into snackbar implementation. It has an internal variable `isActive` that mirrors v-model (your isPasswordResetSuccess)
(~ `watch(v-model) { isActive = value }`).

So that's why notification still disappeared, even though isPasswordResetSuccess=true when it tried to set it to false after 9 seconds.

So sideeffect of `v-snackbar will attempt to set isPasswordResetSuccess = false, which will in turn commit passwordResetSuccess=true to store` isn't that notification doesn't disappear (rescued by isActive), but that it will never appear again (doomed by watch)

:confused: That's a tougher bug to spot